nbs/60_medical.imaging.ipynb

```diff
@@ -80,6 +80,56 @@
     "_all_ = ['DcmDataset', 'DcmTag', 'DcmMultiValue', 'dcmread', 'get_dicom_files']"
```

Comment:
<p>This isn't a complete sentence. Maybe "A <code>dcmread</code>  method is added to <code>Path</code>,  which in turn call's pydicom's <a href="https://pydicom.github.io/pydicom/dev/reference/generated/pydicom.filereader.dcmread.html" target="_blank">dcmread</a> function, for instance:"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2732/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Updated this with better wording</p>